src/containers/Dashboard/DashboardStyles.ts

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from '@material-ui/core';
+
+const drawerWidth = 240;
+
+export const useDashboardStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex'
+  },
+  toolbar: {
+    paddingRight: 24 // keep right padding when drawer closed
```

Comment:
Use `theme.spacing()`, the unit is 8px by default

Reply:
Sure, let's say No to hardcode

Done